deploy/rbd/kubernetes/csi-provisioner-rbac.yaml

```diff
@@ -86,9 +86,6 @@ metadata:
   namespace: default
   name: rbd-external-provisioner-cfg
 rules:
-  - apiGroups: [""]
```

Comment:
rbd can use a configmap for KMS settings?

Reply:
hmmm.. let me revisit this part then :)